Refactored FizzBuzz to include two arrays.

anthonyfalzetti opened the pull request:
[#88972952]

anthonyfalzetti committed:
Refactored FizzBuzz to include two arrays.

[#88972952]

jasonnoble commented on arrays/fizz_buzz_array.rb:
```diff
@@ -0,0 +1,28 @@
+
+numbers = (1..100).to_a
+fizz_buzz_array = numbers.map do |number|
+  if number % 3 == 0 && number % 5 == 0
```
![duplicate-content](https://cloud.githubusercontent.com/assets/22501/6365755/b745a424-bc79-11e4-9b91-e9015e9da40e.jpg)

anthonyfalzetti committed:
Create two seperate arrays for this homework assignment.

Is the numbers array the other is the numbers array after going through the fizzbuzz method.

[#88972952]
